drivers/video/fbdev/bcm2708_fb.c

```diff
@@ -267,14 +338,16 @@ static int bcm2708_fb_check_var(struct fb_var_screeninfo *var,
 
 static int bcm2708_fb_set_par(struct fb_info *info)
 {
+	static int old_firmware;
+
 	struct bcm2708_fb *fb = to_bcm2708(info);
 	struct fb_alloc_tags fbinfo = {
 		.tag1 = { RPI_FIRMWARE_FRAMEBUFFER_SET_PHYSICAL_WIDTH_HEIGHT,
-			  8, 0, },
+			8, 0, },
```

Comment:
Why the whitespace change? The original looks correct in that the 8 lines up with RPI_FIRMWARE_....